### DISCUSSION TITLE
Beta badge with or without icon

### DISCUSSION THREAD

**Discussion by char-mainewang** [twilio-labs/paste#3453] [Q&A] [answered]
Hey team, I was wondering if it's standard across Twilio apps to use the New badge with icon for products in Beta and Pilot phases. It's recommended to include it on the [New Badge component](https://paste.twilio.design/components/badge#new-badge) page, but the beta badges on the [side bar navigation](https://paste.twilio.design/components/sidebar-navigation#about-sidebar-navigation) doesn't have it. Do y'all have recommendations for when to not include the icon?

**Comment by serifluous** [comment, marked as answer]
No catch-all recommendations from us unfortunately—It's going to be mostly use-case based. Like, would the icon reinforce the meaning or intent of the badge, or does it take away from it? Are multiple badges being used, and does including an icon reinforce similarities between badges, especially if there's different text content in each badge?

The sidebar beta badge doesn't have it, for example, because the space in sidebar is pretty compact, and one of its primary purposes is to let customers know the feature is experimental, rather than market its newness. 

Let us know if there's standard practice y'all are working with in Segment! Happy to add more detail to our docs based on more use cases.

**Reply by char-mainewang** [reply]
That's super helpful, I agree that it's not necessary to include the icon in the sidebar as it's already pretty compact. 

The Segment team does have a few relevant docs / figma files for in-app product release communication
-  an internal [Pilot/Beta Legal Guidance and Requirements doc](https://wiki.hq.twilio.com/pages/viewpage.action?pageId=453904614) that outlines the necessary design pattern for features in those release phaese
- the first pilot/beta design library to pair with the above guidance with some pointers for where the badge should go in different situations (created by Kelly)
- the latest guidance in the [Segment Design Library](https://www.figma.com/file/4tOxLhDIMO2MYvF925RQKc/Segment-Design-Library?type=design&node-id=36-34435&mode=design&t=lRlQ8uetZvDTAAZv-11) (created by Brandon)

I'm leaning towards not using the badge for pilot and beta in the sidebar, but using it else where such as the modal and feature/product page.

**Reply by serifluous** [reply]
Sounds great! Thanks for the links—I'm going to send them to Brand, too, because it seems like they were working on pilot/beta stage badges for our marketing website recently, too.
